dolphin/service.py

```diff
@@ -64,13 +63,13 @@
                 help='Wraps the socket in a SSL context if True is set. '
                      'A certificate file and key file must be specified.'),
     cfg.HostAddressOpt('trap_receiver_address',
-                       default=constants.DEF_TRAP_RECV_ADDR,
```

Comment:
It seemed that these constants would not be used anywhere else, so I think it should be removed from constants.py